pkg/kritis/admission/admission.go

```diff
@@ -15,3 +15,43 @@ limitations under the License.
 */
 
 package admission
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/grafeas/kritis/pkg/kritis/admission/constants"
+	"k8s.io/api/admission/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// This is very basic admission controller which allows all pods.
+func AdmissionReviewHandler(w http.ResponseWriter, r *http.Request) {
+	// TODO: Add image security policy check
+	// 1. Get all Image security Polices
+	// 2. Get all pod specs
+	// 3. For all pods Get Vulnerabilities for images
+	// 4. Validate policy
+	// 5. Attest.
+	status := &v1beta1.AdmissionResponse{
+		Allowed: true,
+		Result: &metav1.Status{
+			Status:  string(constants.SuccessStatus),
+			Message: constants.SuccessMessage,
+		},
+	}
+	WriteHttpResponse(status, w)
+}
+
+func WriteHttpResponse(status *v1beta1.AdmissionResponse, w http.ResponseWriter) {
```

Comment:
nit: catch the errors and return them here.

Reply:
umm the error will be written to the Http Response object. Even if i return the error to the func AdmissionReviewHandler, 
there is no use. At the most we can log it.

Reply:
Sorry I mean there can be an actual error returned from the w.Write call at the bottom. We should return that and log it.